.github/workflows/ci.yml

```diff
@@ -138,48 +138,3 @@ jobs:
         run: |
           echo ${{ needs.compatibility-test.result }}
           test ${{ needs.compatibility-test.result }} == "success"
-
-  integration:
```

Comment:
We will need to bring this back, as it runs tests for every instrumentation.

Reply:
I just reverted the change. What do you want to do about the test matrix: https://github.com/open-telemetry/opentelemetry-go-contrib/blob/bb3cf243ee89f724bcab1da580323cbe775505da/.github/workflows/ci.yml#L145

Reply:
Ah, my bad, you're right. I read this step too quickly, and didn't see it was only for the mongo driver.
Then, yes, it needs to be removed.